tox.ini

```diff
@@ -23,21 +23,22 @@ extras =
     py313: jax,dev,matplotlib,dask
 deps =
     pytest-cov
+    pytest-xdist
```

Comment:
But then shouldn't this be listed in the `pyproject.toml` instead?

Reply:
I'm not sure I see significance to moving it there, but I can definitely do so: developers can decide if they want to use this locally or not. The main change is to use it on the CI.